brianlong/www.validators.app
# 187184054 sort user ping stats

#### What's this PR do?
- DESCRIBE THE PURPOSE OF THE PR HERE

#### How should this be manually tested?
- run `rails r dev/create_test_ping_things.rb` a couple of times, and then `rails r "PingThingUserStatsWorker.perform_async('mainnet')"`
- review ping-thing page and confirm that stats are sorted by 60-min avg latency

#### What are the relevant tickets?
- URL to the PivotalTracker ticket

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
